packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

```diff
@@ -0,0 +1,305 @@
+
+### Step 2: Code Review
+
+* **Pending review by Okta** &mdash; the Okta OIN team is now conducting internal QA tests and will notify you by email when the QA review is complete. If the QA test is successful, your submission is published automatically in the OIN.
+* **Action required** &mdash; Okta has found QA issues that require your correction. Check your email for results from the Okta QA review. Make the requested changes as an update to your existing submission.
+* **Final review by Okta** &mdash; the Okta OIN team is conducting a final internal QA test based on previously requested changes and will notify you by email when the final QA review is complete. If the review is successful, your submission is published automatically in the OIN.
+
+### Step 3: Published
+
+* Congratulations, your integration is published in the OIN!
+
+The following flowchart outlines the entire process:
+
+![ISV Submission process flow](/img/oin/isv-portal_submission_flow.png "Process flow diagram for Okta and ISV submission")
+
+## Update your published integration
+
+If you need to make protocol changes to your integration that is already published in the OIN catalog, you can visit the [OIN Manager](https://oinmanager.okta.com/) and create an updated version of the integration.
+
+Similarly, when you enable a new capability in your application (for example, adding SCIM provisioning onto an existing published SAML application), you don't need to create an entirely new submission. You can update your existing submission to enable and specify the settings for that protocol, then submit the updated integration.
+
+1. Sign on to the OIN Manager using the credentials for your original dev org.
+    >**Note:** You must submit the updated integration using the same dev org that was used to make the original submission, otherwise the Okta OIN team rejects the update.
+1. The published integration appears on your integrations page. Click **Update**.
+
+    This creates a new instance of your integration where you can safely change any of the parameters. Your existing integration remains in the OIN catalog and keeps the previous settings until this new version is published.
+1. Update any of the parameters for your existing protocols, or add a new protocol depending on your needs.
+
+    If you need to leave your in-progress submission at any point, you can return to it through the OIN Manager. When you sign on again, you will see the published version and your in-progress version. Click **Edit** on the in-progress version to pick up where you left off.
+1. When you have completed the updates, or have filled the new protocol information so that indicator shows 100% complete, you can click **Submit for Review**.
```

Comment:
```suggestion
1. When you complete the updates or fill in the new protocol information so that the indicator shows 100% complete, you can click **Submit for Review**.
```